src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$column-width: 100px;
+$gap: 1px;
```

Comment:
Variables should be defined at the beginning of the SCSS file or in a separate `_variables.scss` file if the project structure allows for it, to improve maintainability and reusability.